app/clients/cbc_proxy.py

```diff
@@ -0,0 +1,80 @@
+import json
+
+import boto3
+
+
+# Noop = no operation
+class CBCProxyNoopClient:
+
+    def init_app(self, app):
+        pass
+
+    def create_and_send_broadcast(
+        self,
+        identifier, headline, description,
+    ):
+        # identifier=broadcast_message.identifier,
+        # headline="GOV.UK Notify Broadcast",
+        # description=broadcast_message.description,
+        pass
+
+    # We have not implementated updating a broadcast
+    def update_and_send_broadcast(
```

Comment:
Optional:

I'm not too sure on the method names chosen: `create_and_send_broadcast`, `update_and_send_broadcast` and `cancel_broadcast`. 

Each one of them ends up doing the same thing which is sending a broadcast event to the CBC proxy I believe? If so it would be more consistent to have `create_and_send_broadcast_event`, `update_and_send_broadcast_event`, `cancel_and_send_broadcast_event` but in that case I'd be tempted to keep it simpler as `create_broadcast`, `update_broadcast`, `cancel_broadcast`.

I'm not too sure on my suggestions though as my mental is still quite immature on the whole broadcast process so have a read and feel free to make a decision based on what you see fit :)

Reply:
We started off with `create_broadcast`, `update_broadcast`, and `cancel_broadcast`

However I did not like `create_broadcast` and `update_broadcast` because it seems a bit like an object constructor rather than a _thing-doer_ which is why the `_and_send_` infix is present